diff/schema_list_diff.go

```diff
@@ -13,9 +15,11 @@ The result is a combination of two diffs:
 2. Diff of schemas without a $ref (inline schemas): number of added/deleted schemas; modified=only if exactly one schema was added and one deleted, the Modified field will show a diff between them
 */
 type SchemaListDiff struct {
-	Added    int             `json:"added,omitempty" yaml:"added,omitempty"`
-	Deleted  int             `json:"deleted,omitempty" yaml:"deleted,omitempty"`
-	Modified ModifiedSchemas `json:"modified,omitempty" yaml:"modified,omitempty"`
+	Added          int              `json:"added,omitempty" yaml:"added,omitempty"`
+	AddedSchemas   utils.StringList `json:"-" yaml:"-"`
```

Comment:
Include this in the output, it will help the users who are performing a diff to see exactly what was added/deleted